packages/backend/src/core/SyncScheduler.ts

```diff
@@ -23,31 +23,38 @@ export class SyncScheduler {
   )
 
   async start() {
-    let last = await this.statusRepository.getLastBlockNumberSynced()
+    let lastSynced = await this.statusRepository.getLastBlockNumberSynced()
+    const lastKnown = this.blockDownloader.getLastKnownBlock()
 
-    this.logger.info('Last block number synced', { last })
+    this.logger.info('start', { lastSynced, lastKnown: lastKnown.number })
 
-    await this.dataSyncService.revert(last)
+    await this.dataSyncService.discard({ from: lastSynced })
```

Comment:
I think that replacing `discard({ from })` with `discardAfter(block)` improves the readability.

Reply:
I went with `discardFrom` to avoid changing all places wih `- 1`. LMK if that's ok.